hw/ip/dma/data/dma_testplan.hjson

```diff
@@ -316,7 +316,7 @@
               - Check if DMA generates an alert
               '''
         stage: V2S
-        tests: ["dma_illegal_addr_range"]
+        tests: ["dma_generic_stress", "dma_handshake_stress"]
```

Comment:
@alees24: Could you please confirm that the `dma_generic_stress` and `dma_handshake_stress` tests include random configurations that exercise illegal source or destination addresses?

Reply:
Yes, they do, but 'dma_mem_enabled' is better at it; another case of a more directed sequence to increase probability.